components/brave_rewards/resources/extension/brave_rewards/background/reducers/rewards_panel_reducer.ts

```diff
@@ -37,6 +37,14 @@ const updateBadgeTextAllWindows = (windows: chrome.windows.Window[], state?: Rew
 
 }
 
+const handledByGreaselion = (url: URL) => {
```

Comment:
please log an issue to remove this when all platforms migrate

Reply:
https://github.com/brave/brave-browser/issues/11403